src/main/java/de/retest/recheck/review/ignore/ElementAttributeShouldIgnore.java

```diff
@@ -48,15 +52,14 @@ public ElementAttributeShouldIgnoreLoader() {
 		@Override
 		protected ElementAttributeShouldIgnore load( final MatchResult regex ) {
 			final String matcher = regex.group( 1 );
-			final Loader<Matcher> loader = Loaders.get( ElementRetestIdMatcher.class );
+			final Loader<Matcher> loader = Loaders.get( matcher );
 			final String key = regex.group( 2 );
 			return new ElementAttributeShouldIgnore( loader.load( matcher ), key );
 		}
 
 		@Override
 		public String save( final ElementAttributeShouldIgnore ignore ) {
-			final Loader<Matcher> loader = Loaders.get( ElementRetestIdMatcher.class );
-			return String.format( FORMAT, loader.save( ignore.matcher ), ignore.key );
+			return ignore.toString();
```

Comment:
Maybe let the `RegexLoader` default to `toString`?

Reply:
Don't understand what you mean. Separate branch for that?

Reply:
The `RegexLoader#save` can be implemented to default to `ignore.toString()` so you don't have to overwrite this every time.